elpy-shell.el

```diff
@@ -1259,19 +1260,31 @@ With a prefix argument, ignore the existing breakpoints."
         (error "Debugging only work for buffers visiting a file")
       (elpy-shell--ensure-shell-running)
       (save-buffer)
-      (let ((bp-lines (elpy-pdb--get-breakpoint-positions)))
-        (if (or arg (= 0 (length bp-lines)))
-            (progn
-              (elpy-pdb--refresh-breakpoints '())
-              (elpy-pdb--start-pdb t))
-          (elpy-pdb--refresh-breakpoints bp-lines)
-          (elpy-pdb--start-pdb)
-          (python-shell-send-string "continue")))
+      (let (buffer-breakpoint-lst)
+	(dolist (elt (buffer-list))
+	  (with-current-buffer elt
+	    (if (string= major-mode "python-mode")
+		(progn
+		  (dolist (breakpt (elpy-pdb--get-breakpoint-positions))
+		    (setq buffer-breakpoint-lst (cons (list breakpt buffer-file-name) buffer-breakpoint-lst))
+	   	    )
+		  )
+	      )
+	    )
+	  )
```

Comment:
Nit: Can we put all these parens in the same line? It'll conform to the rest of format in the file. Thanks!